From b2bed44b42451bc94c41a10c51136c65a4d5a6ea Mon Sep 17 00:00:00 2001
From: Joerg Hofrichter <joerg.hofrichter@ni.com>
Date: Thu, 30 May 2024 09:27:10 +0200
Subject: [PATCH] mpm: usrp_update_fs: correct outdated mender command in
 prints

'mender -commit' is outdated, 'mender commit' needs to be used instead.
---
 mpm/python/usrp_update_fs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mpm/python/usrp_update_fs b/mpm/python/usrp_update_fs
index 45714f2c0..834a14625 100755
--- a/mpm/python/usrp_update_fs
+++ b/mpm/python/usrp_update_fs
@@ -165,7 +165,7 @@ def run():
         return False
     apply_image(mender_image)
     print("Applied image. After reboot, check if everything works, and then "
-          "run the command '$ mender -commit' to confirm (otherwise, this "
+          "run the command '$ mender commit' to confirm (otherwise, this "
           "update will be undone).")
     print("Note: Any data stored in this partition will be not accessible "
           "after reboot.")
-- 
2.34.1

